client/views/helpers/nav/nav.js

```diff
@@ -35,7 +35,12 @@ Template.nav.events({
       Blaze.remove(popoverTemplate);
     });
   },
+  'click #navCode': function(event, template) {
+    const parentNode = document.getElementById('nav');
+    popoverTemplate = Blaze.renderWithData(Template.qr_code, {
+      link: `https://questions.dev.berkmancenter.org/list/${template.data.slug}`
```

Comment:
@singhsanket143 can you replace the hardcoded domain name with some preconfigured environment variable?

Reply:
@peter-hank Updated the domain name